files/qsub.pl

```diff
@@ -573,19 +583,26 @@ sub parse_script
             }
         }
     };
+
     # if -j PBS directive is in the script,
     # do not use default error path for slurm
     my @check_eo = qw(e o);
-    if ($set{'j'}) {
+    if ($set{'j'} || $orig_args{'-j'}) {
```

Comment:
this should not be needed here.
the defaults is already handled with line below, and `make_command` should only set defaults, not the actual `-e` command.